src/adapters/givPowerBalanceAggregator/givPowerBalanceAggregatorAdapter.ts

```diff
@@ -0,0 +1,108 @@
+// @ts-ignore
+
+import axios from 'axios';
+import {
+  GetBalanceOfAddressesInputParams,
+  GetBalanceOfAddressesResponse,
+  GetBalanceOfAnAddressesResponse,
+  GetBalancesUpdatedAfterASpecificDateInputParams,
+  GetBalancesUpdatedAfterASpecificDateResponse,
+  GetLatestBalanceOfAnAddressInputParams,
+  GetLeastIndexedBlockTimeStampInputParams,
+  GivPowerBalanceAggregatorInterface,
+} from './givPowerBalanceAggregatorInterface';
+import { logger } from '../../utils/logger';
+
+const formatResponse = (balance: {
+  address: string;
+  balance: string;
+  update_at: Date;
+  networks: number[];
+}): GetBalanceOfAnAddressesResponse => {
+  return {
+    address: balance.address,
+
+    // TODO convert wei to eth
+    balance: Number(balance.balance),
```

Comment:
The number cannot hold the balance in wei, it doesn't have enough precision!
We can do the same as we used to do on fetching balance from the subgraph directly.

Reply:
Done